src/jolokia/context.ts

```diff
@@ -0,0 +1,19 @@
+import { createContext } from 'react';
+
+export type jolokiaLoginSource = 'api' | 'session';
+
+export type JolokiaLogin = {
+  isSucces: boolean;
```

Comment:
Currently, the boolean variable is spelled as `isSucces` in several places. Could you please update the spelling to `isSuccess` for clarity and correctness? This change will help to maintain consistency and readability in the codebase.